test/unit/controllers/StatementControllerSpec.scala

```diff
@@ -34,16 +34,19 @@ import play.api.inject.guice.GuiceApplicationBuilder
 import play.api.mvc.Cookie
 
 class StatementControllerSpec extends WordSpec with Matchers with MockitoSugar with GuiceOneAppPerSuite {
-  private val fakeRequest = FakeRequest("GET", "/")
+  private val fakeRequest  = FakeRequest("GET", "/")
```

Comment:
I'm nervous about lining up the equals on subsequent lines - it will lead to a history which will be harder to read because it will contain line changes for formatting.

Can we update the rules to have just a single space before & after the equals?

Reply:
These are the rules used in all our other repositories, isn't consistency more important?